### PR TITLE
Pay 4983 status history

### PR DESCRIPTION
**Pull request by sayali-salunkhe**

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/refunds/dtos/responses/StatusHistoryDto.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+@Builder(builderMethodName = "buildStatusHistoryDtoWith")
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+@JsonInclude(NON_NULL)
+public class StatusHistoryDto implements Comparable<StatusHistoryDto> {
```
Comment (amitchidrewar1301) [inline src/main/java/uk/gov/hmcts/reform/refunds/dtos/responses/StatusHistoryDto.java:19]: As discussed can you please use Comparator.Comparing  of streams for sorting based on date created?

Comment (sayali-salunkhe) [reply src/main/java/uk/gov/hmcts/reform/refunds/dtos/responses/StatusHistoryDto.java]: Done

File: src/main/java/uk/gov/hmcts/reform/refunds/dtos/responses/UserIdentityDataDto.java [hunk 2]
```diff
@@ -1,14 +1,12 @@
 package uk.gov.hmcts.reform.refunds.dtos.responses;
 
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
+import lombok.*;
```
Comment (vigneshpushkaranHMCTS) [inline src/main/java/uk/gov/hmcts/reform/refunds/dtos/responses/UserIdentityDataDto.java:3]: revert with individual import

File: src/main/java/uk/gov/hmcts/reform/refunds/services/RefundsServiceImpl.java [hunk 3]
```diff
@@ -9,24 +9,24 @@
...
 import uk.gov.hmcts.reform.refunds.utils.ReferenceUtil;
 import uk.gov.hmcts.reform.refunds.utils.StateUtil;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```
Comment (vigneshpushkaranHMCTS) [inline src/main/java/uk/gov/hmcts/reform/refunds/services/RefundsServiceImpl.java:29]: replace withy individual import

File: src/main/java/uk/gov/hmcts/reform/refunds/dtos/responses/UserIdentityDataDto.java (hunk 2 above)
Comment (sayali-salunkhe) [reply src/main/java/uk/gov/hmcts/reform/refunds/dtos/responses/UserIdentityDataDto.java]: Done

File: src/main/java/uk/gov/hmcts/reform/refunds/services/RefundsServiceImpl.java (hunk 3 above)
Comment (sayali-salunkhe) [reply src/main/java/uk/gov/hmcts/reform/refunds/services/RefundsServiceImpl.java]: Done

**Review by vigneshpushkaranHMCTS — APPROVED** [review]

**Pull request merged**